org-journal.el

```diff
@@ -327,63 +365,66 @@ Whenever a journal entry is created the
     (let* ((entry-path (org-journal-get-entry-path time))
            (should-add-entry-p (not prefix)))
 
-      ;; open journal file
+      ;; open journal fiel
       (unless (string= entry-path (buffer-file-name))
         (funcall org-journal-find-file entry-path))
+
+      (let ((entry-header
```

Comment:
Could you add a short comment here to describe what this is doing?

Reply:
Sure!

Reply:
Oh, I see, this is the "empty file" block, followed by a property.

Reply:
Exactly!